release-notes/7.0/preview/api-diff/preview4/Microsoft.NETCore.App/7.0-preview4_System.Numerics.md

```diff
@@ -0,0 +1,198 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
++    public interface IAdditionOperators<TSelf, TOther, TResult> where TSelf : IAdditionOperators<TSelf, TOther, TResult> {
++        TResult operator +(TSelf left, TOther right);
++        TResult op_CheckedAddition(TSelf left, TOther right);
```

Comment:
The diff tool looks to display a lot of this incorrectly and doesn't correctly handle `static abstracts in interfaces`:
```suggestion
+        static abstract TResult operator +(TSelf left, TOther right);
+        static abstract TResult operator checked +(TSelf left, TOther right);
```

Reply:
Fixed.